src/cli-flags-helper.js

```diff
@@ -0,0 +1,38 @@
+const targetList = new Set([
+  "acorn",
+  "babel",
+  "babylon",
+  "buble",
+  "chai",
+  "coffeescript",
+  "espree",
+  "esprima",
+  "jshint",
+  "lebab",
+  "prepack",
+  "prettier",
+  "source-map",
+  "typescript",
+  "uglify-es",
+  "uglify-js"
+]);
+
+function getOnlyFlag(argv) {
+  if (process.argv.indexOf("--only") != -1) {
+    return process.argv[process.argv.indexOf("--only") + 1];
+  }
+}
+
+module.exports = {
+  getTarget: function() {
```

Comment:
this can be an arrow function

Reply:
Just out of curiosity, is there a reason to be an arrow function? Like, any performance factor?

Reply:
Performance should be on par with regular functions. *summons @bmeurer*

I personally use arrow functions by default, only falling back to regular functions when I explicitly need a non-lexically bound `this`, `arguments`, `super`, or `new.target`.